File: 프로그래머스/unrated/181929. 원소들의 곱과 합/원소들의 곱과 합.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    int answer = 0;
    int sum=0, mul=1;
    for(int i=0;i<num_list.size();i++){
        sum += num_list[i];
        mul *= num_list[i];
    }
    return (sum*sum) > mul ? 1 : 0;
}

File: 프로그래머스/unrated/181929. 원소들의 곱과 합/원소들의 곱과 합.py
def solution(num_list):
    answer = 0
    sum_of_square = sum(num_list)
    sum_of_square = sum_of_square * sum_of_square
    mul = 1
    for num in num_list:
        mul *= num
    return 1 if sum_of_square > mul else 0